pkg/apis/aerogear/v1alpha1/types.go

```diff
@@ -53,13 +53,14 @@ type KeycloakSpec struct {
 
 //KeycloakBackup details of a backup task
 type KeycloakBackup struct {
-	Name                     string `json:"name"`
-	Schedule                 string `json:"schedule"`
-	EncryptionKeySecretName  string `json:"encryption_key_secret_name"`
-	AwsCredentialsSecretName string `json:"aws_credentials_secret_name"`
-	DbCredentialsSecretName  string `json:"db_credentials_secret_name"`
-	Image                    string `json:"image"`
-	ImageTag                 string `json:"image_tag"`
+	Name                     string            `json:"name"`
+	Labels                   map[string]string `json:"labels"`
```

Comment:
It would be good to add the new `labels` property to the schema: https://github.com/integr8ly/keycloak-operator/blob/master/deploy/crds/Keycloak_crd.yaml#L28